src/enums/routeNames.ts

```diff
@@ -25,4 +25,8 @@ export enum RouteNames {
   REGISTRATION_DEFINE_BUSINESS = 'registration-define-business',
   REGISTRATION_PEOPLE_ROLES = 'registration-people-roles',
   REGISTRATION_REVIEW_CONFIRM = 'registration-review-confirm',
+
+  // Entity Dissolution
+  DISSOLUTION_ENTITY= 'dissolution-entity',
```

Comment:
The first part of this name conflicts with the other dissolution filings above.
The second part of this name is too general: every business is an entity.

If this feature specific to Firms? Will it apply to other corp types -- which ones? Will the new ones use the same route / code?

I suggest, maybe, "firm-dissolution" ...?

If you change this, please sync the component name as well.